Source/Core/Core/PowerPC/Jit64Common/Jit64AsmCommon.cpp

```diff
@@ -301,28 +301,28 @@ void QuantizedMemoryRoutines::GenQuantizedStore(bool single, EQuantizeType type,
     }
     else if (quantize > 0)
     {
-      MULSS(XMM0, M(&m_dequantizeTableS[quantize * 2]));
+      MULSS(XMM0, MConst(m_dequantizeTableS, static_cast<size_t>(quantize * 2)));
```

Comment:
Why do you `static_cast` here, but not elsewhere? Something I missed on the GitHub review pages due to context?